### ISSUE TITLE
Add anchors

### ISSUE THREAD

**Issue by cdaniel** [cdaniel/atlas2#181]
- [x] Add users to the map
- [x] Make users draggable
- [x] Make it possible to establish dependencies to other nodes
- [x] Make it possible to delete said dependencies
- [x] Ensure deleted nodes remove appropriate dependencies
- [x] Test the point above (map-schema # removeNode)
- [x] Support for diff
- [x] Make new changes downloadable
- [x] Make it possible to rename users
- [x] Inspect submap forming
- [x] Tests
- [x] Investigate why connection style is changed when diff is enabled.

**Comment by cdaniel** [comment]
Blocked by https://github.com/jsplumb/jsPlumb/issues/670

**Comment by cdaniel** [comment]
Freetext map titles will be tracked in a separate issue.
